hal/targets/hal/TARGET_Freescale/TARGET_KSDK2_MCUS/TARGET_K64F/storage_driver.c

```diff
@@ -578,19 +703,43 @@ static int32_t executeCommand(struct mtd_k64f_data *context)
         /* Execution may result in failure. Check for errors */
         if (failedWithAccessError() || failedWithProtectionError()) {
             clearErrorStatusBits();
+            #if EXISTS_POSSIBILITY_OF_CONCURRENT_READ
+                __enable_irq();
```

Comment:
Really not liking this pattern. It would make far more logical sense, and tidy the code, to enable interrupts immediately after the "while busy" loop. And disable before the launchCommand.